provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -2996,3 +3005,57 @@ spec:
     defaultRequest:
       memory: {{ config.kube_config.aci_containers_memory_request }}
     type: Container
+{% if config.kube_config.use_aci_containers_host_priority_class %}
+---
+{% set flavor_version = config.flavor.split('-') %}
+{% set major_version = flavor_version[1].split('.')[0] | int %}
+{% set minor_version = flavor_version[1].split('.')[1] | int %}
+{% if (major_version >= 4 and  minor_version >= 9
+       and flavor_version[2] in ["openstack", "esx", "baremetal"]) %}
+apiVersion: scheduling.k8s.io/v1
+{% else %}
+apiVersion: scheduling.k8s.io/v1beta1
+{% endif %}
```

Comment:
Can we just do this processing once in the config build stage, set the api version string as a config, and use it here?